migrations/20220809015149-create-program-location.js

```diff
@@ -10,9 +10,23 @@ module.exports = {
       },
       LocationId: {
         type: Sequelize.INTEGER,
+        allowNull: false,
+        references: {
+          model: {
+            tableName: 'Locations',
+          },
+          key: 'id',
+        },
```

Comment:
This additional column configuration in the migration file lets the database set up a FOREIGN KEY relationship between this table and the referencing table. That means that the database will check and make sure that the values in this column actually match id values in the corresponding table.